### PR TITLE
ServiceIPStaticSubrange alpha feature in 1.24

### PR DESCRIPTION
**Pull request by aojea**
/sig network
KEP https://github.com/kubernetes/enhancements/issues/3070

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -201,6 +201,7 @@ different Kubernetes components.
 | `ServiceLBNodePortControl` | `true` | Beta | 1.22 | |
 | `ServiceLoadBalancerClass` | `false` | Alpha | 1.21 | 1.21 |
 | `ServiceLoadBalancerClass` | `true` | Beta | 1.22 | |
+| `ServiceIPStaticSubrange` | `false` | Alpha | 1.24 | |
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:204]: This line should go before line 198.

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 2]
```diff
@@ -1032,6 +1033,9 @@ Each feature gate is designed for enabling/disabling a specific feature:
   topology of the cluster. See
   [ServiceTopology](/docs/concepts/services-networking/service-topology/)
   for more details.
+- `ServiceIPStaticSubrange`: Enable a new strategy for Services ClusterIP allocations, so users can subdivide the ClusterIP range. Dynamic allocated ClusterIP addresses will be allocated preferently from the upper range allowing users to assign static ClusterIPs from the lower range with a low risk of collision. See
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:1036]: Please wrap the long line as we did for other entries.

```suggestion
- `ServiceIPStaticSubrange`: Enable a new strategy for Services ClusterIP allocations,
  so users can subdivide the ClusterIP range. Dynamic allocated ClusterIP addresses
  will be allocated preferently from the upper range allowing users to assign static ClusterIPs
  from the lower range with a low risk of collision. See
```

**Comment by tengqm** [comment]
This should target at the `dev-1.24` branch rather than `main`.
/hold

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 1 above)
Comment (aojea) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:204]: haha, yeah

**Comment by aojea** [comment]
> This should target at the `dev-1.24` branch rather than `main`. /hold

let me reopen a new one on that branch

Thanks

**Comment by nate-double-u** [comment]
You can change the target branch without opening a new PR, you can use the GitHub web UI to change it.

**Pull request closed without merging**